ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -237,8 +237,18 @@ final class EmberClientBuilder[F[_]: Async: Network] private (
   def withoutPushPromiseSupport: EmberClientBuilder[F] =
     copy(pushPromiseSupport = None)
 
+  private val verifyTimeoutRelations: F[Unit] =
+    logger
+      .warn(
```

Comment:
Wouldn't `info` be enough here?`

Reply:
`warn` level seems appropriate to me, given that:
1. The corresponding Blaze logs had level `warn`
2. The logs only fire once on startup, so they don’t risk being excessively noisy (and startup `info` logs are likely to be missed or ignored along with general application startup logging noise)
3. The message itself _is_ a warning that settings have likely been misconfigured

Reply:
Hey @danicheg - any thoughts on this? Would be nice to get this PR over the line